drivers/dma/Kconfig.dma_pl330

```diff
@@ -0,0 +1,9 @@
+# Copyright 2020 Broadcom
+# SPDX-License-Identifier: Apache-2.0
+
+config	DMA_PL330
+	bool
+	prompt "pl330 dma driver"
```

Comment:
nit: the prompt is a sentence, so use "PL330 DMA driver" instead of the lower case version.

Reply:
ack